src/redux/reducers/componentConfig/initialState.js

```diff
@@ -1,3 +1,46 @@
+const embeddingPreviewInitialConfig = {
```

Comment:
where is this spec used? Is it only for plots and tables?

Reply:
It was not used for anything, fixed.